ingestion/src/metadata/ingestion/ometa/mixins/patch_mixin.py

```diff
@@ -213,6 +215,61 @@ def patch_column_description(
 
         return None
 
+    def patch_table_constraints(
+        self,
+        entity_id: Union[str, basic.Uuid],
+        table_constraints: List[TableConstraint],
+    ) -> Optional[T]:
+        """Given an Entity ID, JSON PATCH the table constraints of table
+
+        Args
+            entity_id: ID
+            description: new description to add
+            table_constraints: table constraints to add
+
+        Returns
+            Updated Entity
+        """
+        table: Table = self._validate_instance_description(
```

Comment:
do we know why this is called like this `_validate_instance_description`? I see the method below, but from what I understand it is not validating anything related to description 🤔

```python
    def _validate_instance_description(
        self, entity: Type[T], entity_id: Union[str, basic.Uuid]
    ) -> Optional[T]:
        """
        Validates if we can update a description or not. Will return
        the instance if it can be updated. None otherwise.

        Args
            entity (T): Entity Type
            entity_id: ID
            description: new description to add
            force: if True, we will patch any existing description. Otherwise, we will maintain
                the existing data.
        Returns
            instance to update
        """

        instance = self.get_by_id(entity=entity, entity_id=entity_id, fields=["*"])

        if not instance:
            logger.warning(
                f"Cannot find an instance of '{entity.__class__.__name__}' with id [{str(entity_id)}]."
            )
            return None

        return instance
```

Reply:
umm right the method name is misleading will change that